cassandra_nodetool/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "categories": [
-    "data store"
+    "data store",
+    "log collection"
```

Comment:
Logs are collected by cassandra integration, does it make sense to categorize `cassandra_nodetool` as `log collection` ?

Reply:
Good catch!